README.md

```diff
@@ -26,3 +26,10 @@ commit-analyzer outputfile.txt --output "commits-and-loc-by-date.csv"
 
 This project's license is **MIT**.  The license can be found `LICENSE` in the
 main directory of this repository.
+
+## Configured Options
+
+| Option    | Brief | Meaning                                                  |
+|:---------:|:-----:|:---------------------------------------------------------|
+| `--help`  | `-h`  | Show this help and exit.                                 |
+
```

Comment:
Revert or enrich with further options? :wrench:

Reply:
Definitely enriching. I think it was premature to commit the `--help` flag, but reverting would not solve that, and I'm still in favor of adding the output to the README. 😄

Reply:
:wrench: :heavy_check_mark: